src/components/course/enrollment/components/ToExecutiveEducation2UEnrollment.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Stack } from '@edx/paragon';
+import { EnrollButtonCta } from '../common';
+import { useTrackSearchConversionClickHandler } from '../../data/hooks';
+
+/**
+ * Renders a hyperlink to the ExecutiveEducation2UPage
+ */
+const ToExecutiveEducation2UEnrollment = ({
+  enrollmentUrl,
+}) => {
+  const handleClick = useTrackSearchConversionClickHandler({
```

Comment:
Something else to consider with using `useTrackSearchConversionClickHandler` is that its intent is to track _conversions_, where the conversion here may actually be after successfully clicking "Confirm registration" from the Executive Education 2U enrollment form, and enrolling in the course.

In the current state, this conversion event from the hook makes an assumption that the user will go all the way through the next steps after clicking the enrollment url (e.g., filling out the Data Sharing Consent form), but here and in the `StatefulEnroll` component, we have an opportunity to track legit conversions after a successful enrollment client side, not just clicking to the enrollment url to navigate to the next service.

That said, I might recommend deferring on making sure that event is in place until https://2u-internal.atlassian.net/browse/ENT-7065 to ensure we have enough time to QA the course page cutover on stage/production before a release by end of week, if possible.